build-tools/scripts/XAVersionInfo.targets

```diff
@@ -68,6 +83,10 @@
     </GitBranch>
     <PropertyGroup>
       <XARepositoryName Condition=" '$(XARepositoryName)' == '' ">xamarin-android</XARepositoryName>
+      <!-- See Azure Pipelines predefined variables. -->
+      <_AndroidPackLabel Condition=" '$(SYSTEM_PULLREQUEST_PULLREQUESTNUMBER)' != '' ">ci.pr.gh$(SYSTEM_PULLREQUEST_PULLREQUESTNUMBER).$(PackVersionCommitCount)</_AndroidPackLabel>
+      <_AndroidPackLabel Condition=" '$(_AndroidPackLabel)' == '' ">ci.$(XAVersionBranch).$(PackVersionCommitCount)</_AndroidPackLabel>
```

Comment:
It doesn't look like `$(XAVersionBranch)` is limited to alphanumeric characters.

Reply:
Thanks, I've added a Regex.Replace to handle this case.